base/applications/rapps/lang/ro-RO.rc

```diff
@@ -236,36 +237,36 @@ END
 
 STRINGTABLE
 BEGIN
-    IDS_LICENSE_OPENSOURCE "Sursă Deschisă"
-    IDS_LICENSE_FREEWARE "Freeware"
-    IDS_LICENSE_TRIAL "Trial/Demo"
+    IDS_LICENSE_OPENSOURCE "Sursă deschisă"
+    IDS_LICENSE_FREEWARE "Gratuit"
+    IDS_LICENSE_TRIAL "Variantă de încercare/Demo"
 END
 
 STRINGTABLE
 BEGIN
     IDS_LANGUAGE_AVAILABLE_TRANSLATION "Suportă limba respectivă"
     IDS_LANGUAGE_NO_TRANSLATION "Suportă alte limbi"
-    IDS_LANGUAGE_ENGLISH_TRANSLATION "Suportă limba Engleză"
-    IDS_LANGUAGE_SINGLE "Singura Limbă"
+    IDS_LANGUAGE_ENGLISH_TRANSLATION "Suportă limba engleză"
+    IDS_LANGUAGE_SINGLE "Singura limbă"
     IDS_LANGUAGE_MORE_PLACEHOLDER " (+%d mult)"
     IDS_LANGUAGE_AVAILABLE_PLACEHOLDER " (+%d disponibil)"
 END
 
 STRINGTABLE
 BEGIN
     IDS_DL_DIALOG_DB_DISP "Aplicații baza de date"
-    IDS_DL_DIALOG_DB_DOWNLOAD_DISP "Actualizare baza de date…"
-    IDS_DL_DIALOG_DB_UNOFFICIAL_DOWNLOAD_DISP "Updating Database… (Unofficial source)"
+    IDS_DL_DIALOG_DB_DOWNLOAD_DISP "Actualizare bază de date…"
+    IDS_DL_DIALOG_DB_UNOFFICIAL_DOWNLOAD_DISP "Actualizare bază de date… (Sursă neoficială)"
 END
 
 STRINGTABLE
 BEGIN
-    IDS_CMD_USAGE "Usage: "
-    IDS_CMD_NEED_PACKAGE_NAME "Error: option %1 expects one or more package name.\n"
-    IDS_CMD_NEED_FILE_NAME "Error: option %1 expects a file name.\n"
-    IDS_CMD_NEED_PARAMS "Error: option %1 expects one or more parameters.\n"
-    IDS_CMD_INVALID_OPTION "Error: Unknown or invalid command line option specified.\n"
-    IDS_CMD_FIND_RESULT_FOR "Find result for %1:\n"
-    IDS_CMD_PACKAGE_NOT_FOUND "Failed to find package %1.\n"
-    IDS_CMD_PACKAGE_INFO "Information about package %1:\n"
+    IDS_CMD_USAGE "Utilizare: "
+    IDS_CMD_NEED_PACKAGE_NAME "Eroare: opțiunea %1 așteaptă unul sau mai multe nume de pachete.\n"
+    IDS_CMD_NEED_FILE_NAME "Eroare: opțiunea %1 așteaptă un nume de fișier.\n"
+    IDS_CMD_NEED_PARAMS "Eroare: opțiunea %1 așteaptă unul sau mai mulți parametri.\n"
+    IDS_CMD_INVALID_OPTION "Eroare: opțiune de linie de comandă specificată fie necunoscută, fie nevalidă.\n"
```

Comment:
I find this a little ambiguous. The user should not need to read it multiple times in order to understand it.